server/src/endpoints/minidump.rs

```diff
@@ -66,15 +113,8 @@ where {
         let future = ForwardBody::new(request, max_payload_size)
             .map_err(|_| BadStoreRequest::InvalidMinidump)
             .and_then(move |data| {
-                validate_minidump(&data)?;
-
-                let mut item = Item::new(ItemType::Attachment);
-                item.set_payload(ContentType::OctetStream, data);
-                item.set_filename(MINIDUMP_FILE_NAME);
-                item.set_attachment_type(AttachmentType::Minidump);
-
+                let item = create_minidump_item(data)?;
```

Comment:
_nit_: This change can be reverted again.

Reply:
ok